src/app/index.tsx

```diff
@@ -121,6 +122,7 @@ export function App() {
               />
               <Route component={NotFoundPage} />
             </Switch>
+            <ToastContainer className="tw-w-max" />
```

Comment:
No, it shouldn't, because with `full` it would be stretched to your whole window width, whereas with `content-max`, it adapts to the width of your content.